test/lib/find-scaffolds.js

```diff
@@ -0,0 +1,134 @@
+import test from 'blue-tape';
+import npm from 'npm';
+import {stub, spy} from 'sinon';
+
+
+import findScaffolds from '../../lib/find-scaffolds';
+
+
+const dependencies = {
+  test: {
+    // should skip this one
+    name: 'whatever-random-name',
+    realPath: 'test realpath'
+    // no keywords, should not fail
+  },
+  prefixedName: {
+    // should return, because it matches name = 'cf-*'
+    name: 'cf-test',
+    realPath: 'prefixed realpath',
+    keywords: ['random', 'keywords']
+  },
+  taggedWithSingleTag: {
+    // should return, because it matches tag 'cloverfield-scaffold'
+    name: 'tagged1',
+    realPath: 'tagged1 realpath',
+    keywords: ['cloverfield-scaffold']
+  },
+  taggedWithTwoTags: {
+    // should return, because it matches two tags 'cloverfield' && 'scaffold'
+    name: 'tagged2',
+    realPath: 'tagged2 realpath',
+    keywords: ['cloverfield', 'scaffold']
+  },
+  incorrectlyTagged: {
+    // should skip this because it matches only one tag 'cloverfield', but not 'scaffold'
+    name: 'incorrectly-tagged',
+    realPath: 'incorrectly-tagged realpath',
+    keywords: ['cloverfield', 'something else']
+  }
+};
+
+
+const before = () => {
```

Comment:
I usually do:

```js
// Just for descriptive names / self documenting code
const before = test;
const after = test;
```

Then for stuff that needs to be repeated a lot before and after tests:

```js
const setup = () => somefixtures
const teardown = (fixtures) => dispose(fixtures);
```